requirements.txt

```diff
@@ -1,5 +1,5 @@
-ansible>=2.7.9
-openstacksdk>=0.26.0
+ansible>=2.7.10
```

Comment:
Is 2.7.10 out yet?  Not according to https://github.com/ansible/ansible/releases - this would break any virtualenv deployment until it's available, right?

Reply:
Yes, that's why it's tagged [wip].

Reply:
Maybe it should be tagged [DNM] (Do Not Merge)!

Reply:
It would be the safest thing to do. Or at least, limit updates to patch versions:

`ansible>=2.7.10,<2.8.0`

Reply:
Or perhaps `ansible~=2.7.10` :-)

Reply:
Ansible 2.7.10 has been released!

Reply:
@brtknr What do you think about preventing an upgrade to Ansible 2.8, as proposed in this thread?

Reply:
Done